### ISSUE TITLE
[feature request] follow links in text posts to other posts

### ISSUE THREAD

**Issue by vdmkenny** [michael-lazar/rtv#294]
Hi, I often read multipart stories which have a link in the text with the previous/next part in them.
Is it possible to follow those links and have the posts open in rtv?
If not, this is a feature I would very much enjoy.

**Comment by michael-lazar** [comment]
Unfortunately I don't think this is currently possible. Could you post an example of a story for reference?

**Comment by vdmkenny** [comment]
https://www.reddit.com/r/talesfromtechsupport/comments/51og8u/frantic_friday_part_2_the_franticing/
This is a very common occurrence in multi-part posts, and it's a bit awkward to navigate them with rtv as of now.

**Comment by michael-lazar** [comment]
Note that this is related to https://github.com/michael-lazar/rtv/pull/314, when that PR goes in you should be able to type

```
/comments/51og8u
```

Admittedly it's not as convenient as pressing a shortcut key, but I think it's a good intermediate step.

**Comment by aetherfaerber** [comment]
I just started using rtv and this is the one issue I experienced.

I understand the ability to quickly open comment threads has been added as of [v1.15.0](https://github.com/michael-lazar/rtv/releases/tag/v1.15.0). But after updating via pip I still get 

> Invalid order ‘54og8u'

when using

`/comments/54og8u`


My first attempt to work around the problem was messing with system configurations. Maybe it is possible to have all urls starting with "https://reddit.com" open in rtv via an default-url-scheme-handler from xdg or through mailcap? It should also be possible to set rtv (with flag -l,  see #412 ) as option to open e.g. text/html files, then set $BROWSER to mimeopen -a and just use urlscan.

PS: I mainly run into this reading /r/tldr so that might be a good testing ground.
